libs/server/Custom/ExpandableMap.cs

```diff
@@ -386,39 +386,39 @@ public bool TryGetFirstId(Func<T, bool> predicate, out int id)
     }
 
     /// <summary>
-    /// Extension methods for ConcurrentExpandableMap
+    /// Extension methods for ExpandableMap
```

Comment:
As discussed:
* We need one global CCM (at the storeWrapper level)
* CCMS is a session specific wrapper for the global CCM, that caches when necessary/possible
* All access to CCM from a session (both front-end and back-end) would happen via CCMS and not CCM
* We can unify the map into a single latch-free expandable map
* We can share this latch-free ExpandableMap between CCM and CCMS for scalable and efficient access